tests/expr_and_series/round_test.py

```diff
@@ -8,10 +9,23 @@
 from tests.utils import compare_dicts
 
 
+def round_half_towards_infinity(n: float, decimals: int) -> float:
+    rounded = round(n, decimals)
+    frac, whole = math.modf(n)
+    if frac == 0.5 and rounded == whole:
+        return whole + 1
+    return rounded
+
+
+def round_per_backend(backend: str, n: float, decimals: int) -> float:
+    if backend.startswith(("polars", "pyarrow")):
+        return round_half_towards_infinity(n, decimals)
+    else:
+        return round(n, decimals)
+
+
```

Comment:
@FBruzzesi  would this be too overengineered?

Reply:
Maybe a bit, especially because we are ending up testing something which each library is (should be) testing already.

On a practical level, I would keep the tests as they were, and add pyarrow backend in the Notes for both [series](https://github.com/narwhals-dev/narwhals/blob/e3de9a07df2461e3f1fbff7d54fd8f7d037843ca/narwhals/series.py#L1936) and [expr](https://github.com/narwhals-dev/narwhals/blob/e3de9a07df2461e3f1fbff7d54fd8f7d037843ca/narwhals/expr.py#L1640)

WDYT?

Reply:
Ah yes, I though so 😝
Yep, all makes sense!